ui/apps/platform/src/Containers/Vulnerabilities/utils/vulnerabilityUtils.ts

```diff
@@ -45,3 +46,15 @@ export function getHighestCvssScore(vulnerabilities: { cvss: number; scoreVersio
     });
     return { cvss: topCvss, scoreVersion: topScoreVersion };
 }
+
+export function getEarliestDiscoveredAtTime(
+    vulnerabilities: NonEmptyArray<{ discoveredAtImage: string }>
+): string {
+    let earliestDiscoveredAt = vulnerabilities[0].discoveredAtImage;
+    vulnerabilities.forEach(({ discoveredAtImage }) => {
+        if (discoveredAtImage.localeCompare(earliestDiscoveredAt) < 0) {
```

Comment:
For my understanding:
* Because ISO 8601 date string consists of ASCII codes only, which locale on user system is irrelevant? 
* Is there any reason to prefer the method to `<` operator here, which unlike `sort` method does not need less, greater, or equal distinction?

Reply:
Honestly, not really. I'll swap to the comparison operator as that is a bit more readable.